CHANGELOG.md

```diff
@@ -19,6 +19,12 @@
 
 #### :rocket: New Feature
 
+- Prebuilt binaries are now provided for all major platforms:
```

Comment:
Can you move this down? I've put uncured and untagged on top on purpose.

Reply:
Ah sorry, I thought these were already in the Main New Features anyway.